dsp/modules/hf_client.py

```diff
@@ -198,6 +199,7 @@ def _generate(self, prompt, **kwargs):
                 port=self.port,
                 json=payload,
                 headers=self.headers,
+                **self.http_request_kwargs,
```

Comment:
should this be added for non-chat models as well? [lines 224-230](https://github.com/stanfordnlp/dspy/pull/1250/files#diff-2722ca06eadb8958a6b321a17668697be2f49099d9f9541ab03fb52299bde9d1R229)